tests/test_rrproxy_call.py

```diff
@@ -7,18 +7,18 @@
 
 
 class TestRRPProxyCall(TestRRPProxyBase):
-    def test_call_calls_request_get_with_correct_url_and_parameters(self):
+    def test_call_calls_session_get_with_correct_url_and_parameters(self):
```

Comment:
It think it would be nice to add a test here that makes sure it calls the requests/session depending on the specified parameter. Technically what we're currently doing is only looking if the `RRPProxy.session` variable is set, but not if it's being used (in the init tests)

Reply:
There's a test that makes sure it's set correctly in the init and I just adjusted this one to use the correct variable

Reply:
It makes sure it's set, but it does not make sure this variable is being used is what I meant :) I.e. you could still do/use something else instead of the variable and no test would fail (i.e. if down below you would just always use `requests` nothing would fail)